install/helm/open-match/values-production.yaml

```diff
@@ -173,6 +173,7 @@ redis:
 open-match-core:
   enabled: true
   ignoreListTTL: 60000ms
```

Comment:
may be represent everything in excess of a minute as a minutes value across the yamls? (like you did for this variable in the values.yaml file?

Reply:
Yeah, they should be in sync. Fixed.